README.md

```diff
@@ -1,20 +1,65 @@
 # date-fns upgrade
 
-Support package that helps with the upgrade.
+Support package with helper functions for `date-fns` upgrade from `1.x` to `2.x`
 
-See [date-fns-upgrade-codemod](https://github.com/date-fns/date-fns-upgrade-codemod) for more details.
+Codemod that will help you apply these helpers automatically located here - [date-fns-upgrade-codemod](https://github.com/date-fns/date-fns-upgrade-codemod).
 
 ## Installation
 
-The library is available as an [npm package](https://www.npmjs.com/package/@date-fns/upgrade).
-To install the package run:
-
 ```bash
 npm install @date-fns/upgrade --save
 # or with yarn
 yarn add @date-fns/upgrade
 ```
 
+## Usage
+
+### convertTokens
+
+`convertTokens` is a helper function used for 2nd argument of `format` function to convert date tokens like `YYYY` to new format. See [this post](https://blog.date-fns.org/post/unicode-tokens-in-date-fns-v2-sreatyki91jg) for more details.
+
+Obviously, you can convert tokens manually :)
+
+```diff
++import { convertTokens } from '@date-fns/upgrade/v2'
+
+const formattedDate = format(
+  new Date(),
+- 'YYYY',
++ convertTokens('YYYY'),
+)
+```
+
+### legacyParse
+
+`date-fns@2.x` functions don't accept string as arguments any more (see [CHANGELOG](https://github.com/date-fns/date-fns/blob/master/CHANGELOG.md#changed)), `legacyParse` is used to simplify that transition. See [this post](https://blog.date-fns.org/post/we-cut-date-fns-v2-minimal-build-size-down-to-300-bytes-and-now-its-the-smallest-date-library-18f2nvh2z0yal) for details on "why" this was done.
+
+```diff
```

Comment:
Please add that `legacyParse` uses the v1 algorithm to convert arguments to dates.

Reply:
Mentioned.